lib/model/state.dart

```diff
@@ -78,10 +103,12 @@ class AppState with ChangeNotifier {
     notifyListeners();
   }
 
-  Future<String?> readPrivateHex() async {
-    // read the key and return it
+  Future<void> checkForSavedPrivateKey() async {
+    // read the key
+    var pk = await _readSecretKey();
```

Comment:
Might be good to explicitly call this `privateKey`. Just thinking back to how easy it was to confuse `sk` and `pk` in other libraries.